Tasks/Common/MSBuildHelpers/make.json

```diff
@@ -4,6 +4,10 @@
             {
                 "url": "https://vstsagenttools.blob.core.windows.net/tools/msbuildlogger/3/msbuildlogger.zip",
                 "dest": "./"
+            },
+            {
+                "url": "https://vstsagenttools.blob.core.windows.net/tools/vswhere/1_0_62/vswhere.zip",
```

Comment:
Can we have fwd link which can be consumed by all tasks pointing to latest release?